lib/kaminari/mandatory_ordering/active_record_extension.rb

```diff
@@ -7,14 +7,14 @@ module ActiveRecordExtension
 
       included do
         descendants.each do |klass|
-          klass.send(:prepend, Kaminari::MandatoryOrdering::ActiveRecordModelExtension)
+          klass.send(:include, Kaminari::MandatoryOrdering::ActiveRecordModelExtension) if klass.superclass == ::ActiveRecord::Base
```

Comment:
Will this work in cases where `klass` inherits from `Something` inherits from `ActiveRecord::Base`?